arbos/l1pricing/l1pricing.go

```diff
@@ -68,12 +68,15 @@ const (
 )
 
 const (
-	InitialEquilibrationUnits uint64 = 60 * params.TxDataNonZeroGasEIP2028 * 100000 // one minute at 100000 bytes / sec
-	InitialInertia                   = 10
-	InitialPerUnitReward             = 10
-	InitialPricePerUnitWei           = 50 * params.GWei
+	InitialInertia           = 10
+	InitialPerUnitReward     = 10
+	InitialPricePerUnitWei   = 50 * params.GWei
+	InitialPerBatchGasCostV7 = 141000
 )
 
+var InitialEquilibrationUnitsV0 = 60 * params.TxDataNonZeroGasEIP2028 * 100000 // one minute at 100000 bytes / sec
+var InitialEquilibrationUnitsV7 = big.NewInt(141000)
```

Comment:
This should be at least as large as the max number of data units in a batch, which is something like 16*10000000.  (The L1 pricer aims to price so that any surplus or deficit in collections is eliminated over the next this many data units. So if a batch is larger than this parameter, the pricer may overshoot.)

Reply:
This was from an earlier commit -- it's now `160000000` as on Nova